data/locale/ar.ts

```diff
@@ -6361,7 +6361,7 @@ Please make sure you have write permission to the file and the directory contain
     </message>
 </context>
 <context>
-    <name>InstrumentMiscView</name>
+    <name>InstrumentTuningView</name>
```

Comment:
As I remember it Transifex's invisible elves will sort it out when you change a variable name. In this case a Class name. Are you sure you need to change every localization manually? I'm pretty sure the changes in InstrumentTrack is enough and that anything in **data/locale/** should be left alone.

Reply:
I'm not sure how good those elves are, but if they do not recognize the change and all strings in the dialog would have to be re-translated, it is probably not that big of a deal. Transifex automatically recommends similar previously translated strings, and there will be a lot to update before the 1.3 release anyway, so somebody will probably fix that in a few seconds as they go through dozens of other strings.

What I do _not_ know, is how often we push the Transifex changes to the LMMS repository. Given how long the release cycle became, I wanted to make sure the existing locales in repo keep working in the mean time (in case the push is done just once before each release).

Reply:
@tresf Is more more into this. I just fear the changes will may be wiped out the next time Transifex is updated. I'm not sure how often the strings are updated. It's so not 2020+ to not have this done by a drone. :)

Reply:
> I just fear the changes will may be wiped out the next time Transifex is updated.

That's what I'm counting on. :) It was meant as a temporary fix before the update from Transifex actually happens. See discussion with @allejok96 (who suggested it) [above](https://github.com/LMMS/lmms/pull/6206#issuecomment-1315873324)

Reply:
Right. I gracefully skipped over that part of the PR history. I'm personally fine with this but I'm not an authority of any kind.

Reply:
> @tresf Is more more into this. I just fear the changes will may be wiped out the next time Transifex is updated. I'm not sure how often the strings are updated. It's so not 2020+ to not have this done by a drone. :)

I don't know that much about Transiflex.  Perhaps you meant to tag @liushuyu ?